<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileViewer</class>
 <widget class="QWidget" name="FileViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="view_Font">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Font</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="fontEdit">
          <property name="text">
           <string>Sortez les cannons, chargez les fusils photoniques, le premier qui doute je le jette aux abysses !!

Khaal'Agan lors de l'attaque du secteur V19876435210W</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="view_Shader">
       <property name="title">
        <string>Shader</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QOpenGLWidget" name="openGLWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="view_Texture">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Texture</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="img_Texture">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLabel" name="lbl_property">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="view_Sound">
       <property name="title">
        <string>Son</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0" colspan="3">
         <widget class="QCommandLinkButton" name="btn_Play">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lbl_soundDuration">
          <property name="text">
           <string>Dur√©e : hh:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>5</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="lbl_soundPlayerStatus">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
